rflx/pyrflx/typevalue.py

```diff
@@ -518,12 +518,14 @@ def __init__(
         model: Message,
         refinements: ty.Sequence["RefinementValue"] = None,
         skip_verification: bool = False,
+        parameters: ty.Mapping[Name, Expr] = None,
         state: "MessageValue.State" = None,
     ) -> None:
         super().__init__(model)
         self._skip_verification = skip_verification
         self._refinements = refinements or []
         self._path: ty.List[Link] = []
+        self.__parameters = parameters or {}
```

Comment:
Naming the attribute `_parameters` would be more consistent to the existing attributes.

Reply:
Fixed in https://github.com/Componolit/RecordFlux/pull/778/commits/2329a3b4337b27bb9714294e02344ff2c60fc622.